src/Doctrine/Orm/RedirectRoute.php

```diff
@@ -0,0 +1,98 @@
+<?php
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2017 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Cmf\Bundle\RoutingBundle\Doctrine\Orm;
+
+use Symfony\Cmf\Bundle\RoutingBundle\Model\RedirectRoute as RedirectRouteModel;
+
+/**
+ * {@inheritdoc}
+ *
+ * This extends the RedirectRoute Model.
+ */
+class RedirectRoute extends RedirectRouteModel
+{
+    /**
+     * Unique id of this route.
+     *
+     * @var integer
+     */
+    protected $id;
+
+    /**
+     * @var string
+     */
+    protected $serialisedParameters;
+
+    /**
+     * Overwrite to be able to create route without pattern.
+     *
+     * Additional options:
+     *
+     * * add_trailing_slash: When set, a trailing slash is appended to the route
+     *
+     * @param array $options
+     */
+    public function __construct(array $options = [])
+    {
+        parent::__construct($options);
+    }
+
+    public function setId($id)
+    {
+        $this->id = $id;
+
+        return $this;
+    }
+
+    public function getId()
+    {
+        return $this->id;
+    }
+
+    /**
+     * Set the parameters for building this route. Used with both route name
+     * and target route document.
+     */
+    public function setParameters(array $parameters)
+    {
+        $this->serialisedParameters = json_encode($parameters);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getParameters()
+    {
+        $params = json_decode($this->serialisedParameters);
+        return is_array($params) ? $params : []; 
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getPath()
+    {
+        $pattern = parent::getPath();
+        if ($this->getOption('add_trailing_slash') && '/' !== $pattern[strlen($pattern) - 1]) {
```

Comment:
do we need this behaviour? i think we should not have the add_trailing_slash option for orm. its needed in phpcr because the repository path is like a directory path, and never has a trailing slash. but in orm, we store a path and can just store the path with `/` at the end.

Reply:
actually, do we need getPath at all? thats used when the redirect route is treated as a route of its own. with your approach, the redirect information is not really a route. maybe we should also call the class just `Redirect` instead of saying something about `Route` when its not a route.

Reply:
Regarding the trailing slash I added this because it otherwise crashed. I didn't have time to research the root cause :/  I could try to remove the getpath() completely.

Regarding the naming, you might be right. As the Redirect is a "content" which works together with a real route object. Will revisit this.

Reply:
actually thats not from the interfaces but from the symfony route class, which we do need to extend for smyfony to consider our thing a route.